pkg/storaged/mdraid-details.jsx

```diff
@@ -208,6 +191,24 @@ export class MDRaidDetails extends React.Component {
             );
         }
 
+        function fix_bitmap() {
+            return mdraid.SetBitmapLocation(utils.encode_filename("internal"), { });
+        }
+
+        let bitmap_message = null;
+        if (mdraid.Level != "raid0" &&
+            client.mdraids_members[mdraid.path].some(m => m.Size > 100 * 1024 * 1024 * 1024) &&
+            mdraid.BitmapLocation && utils.decode_filename(mdraid.BitmapLocation) == "none") {
+            bitmap_message = (
+                <Alert isInline variant="warning"
+                       title={_("This RAID array has no bitmap.  A bitmap can reduce sychronization times significantly.")}>
```

Comment:
I've run RAID in the past but have never heard of a bitmap should we link to some docs explaining what it is?

Reply:
Yeah, we should at least call it "write-intent bitmap".  I am hesitant to put links into Cockpit code... they will all rot away eventually.  But [this](https://raid.wiki.kernel.org/index.php/Write-intent_bitmap) seems to be the canonical one, no?